modules/deploy/partials/linux/install-fips.adoc

```diff
@@ -29,9 +34,32 @@ NOTE: Alternatively, you could run `sudo apt install -y redpanda-fips`, which al
 
 If you wish to only use `rpk` on a FIPS host, run:
 
+[,bash]
+----
+sudo yum install -y repdanda-fips redpanda-rpk-fips
```

Comment:
I noticed below that the Debian installation is `redpanda-rpk-fips` and then `redpanda-fips`.  Here it's reversed.  Neither is wrong, but maybe maintain the same order between both commands?